scripts/mkcloud

```diff
@@ -128,6 +128,9 @@ scp="scp $sshopts"
 : ${install_chef_suse_override:=./install-chef-suse.sh}
 : ${cct_tests:="features:base"}
 
+# Replace special 'jumbo' with numeric so we can compare it later on
+[[ ${want_mtu_size} == jumbo ]] && want_mtu_size=9000
```

Comment:
but want_mtu_size is sourced and exported later on so this is visible on qa_crowbarsetup as well.